src/Akka.Streams.Kafka.Tests/KafkaIntegrationTests.cs

```diff
@@ -150,18 +150,16 @@ protected async Task GivenInitializedTopic(TopicPartition topicPartition)
 
         private static Config Default()
         {
-            var defaultSettings =
-                ConfigurationFactory.FromResource<ConsumerSettings<object, object>>("Akka.Streams.Kafka.reference.conf");
-            
-            var config = ConfigurationFactory.ParseString("akka.loglevel = DEBUG");
+            //var config = ConfigurationFactory.ParseString("akka.loglevel = DEBUG");
+            var config = ConfigurationFactory.ParseString("akka{}");
```

Comment:
Why not to keep Debug level here? I guess this is helpful for debugging by default.